README.md

```diff
@@ -34,3 +34,29 @@ Optional commands are parsed in the configuration module, but not implemented!
 ```
 cargo test
 ```
+
+## Logger
+### Usage
+The logger is configured via environment variables ```TERM_LOG_LEVEL``` and ```FILE_LOG_LEVEL```. ```error!``` macro is the highest-priority and ```trace!``` the lowest, if no log level is provided if defaults to ```LevelFilter::Warn``` for the terminal and ```LevelFilter::Info``` for the log file.
+
+```
+TERM_LOG_LEVEL=<LOG_LEVEL> FILE_LOG_LEVEL=<LOG_LEVEL> ./target/debug/tidybee-agent
+```
+
+### [Log macros](https://docs.rs/log/0.4.20/log/#macros) conventions
+```rust
+// log states of variables
+debug!("server_port: {}", server_port);
+
+// log features as they are executed
+trace!("Starting server");
+
+// log successfully executed features
+info!("Server started");
+
+// log non-fatal behaviours
+warn!("Cannot start the server, retrying");
+
+// log fatal errors
+error!("Could not start the server");
```

Comment:
Can u please put macro in highest priority order ? starting with error! .... trace! thanks